src/tests/ftest/io/dfuse_sparse_file.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import subprocess
+from getpass import getuser
+import paramiko
+import general_utils
+
+from ClusterShell.NodeSet import NodeSet
+from command_utils import CommandFailure
+from ior_test_base import IorTestBase
+
+
+class DfuseSparseFile(IorTestBase):
+    """Dfuse Sparse File base class
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a DfuseSparseFile object."""
+        super(DfuseSparseFile, self).__init__(*args, **kwargs)
+        self.space_before = None
+        self.ret = None
+
+    def tearDown(self):
+        """Tear down each test case."""
+        try:
+            if self.dfuse:
+                self.dfuse.stop()
+        finally:
+            # Stop the servers and agents
+            super(DfuseSparseFile, self).tearDown()
+
+    def get_remote_file_size(self, file_name):
+        """Obtain remote file size.
+
+          Args:
+            file_name (str): name of remote file
+
+          Returns:
+            integer value of file size
+        """
+
+        cmd = "ssh" " {}@{}" " stat -c%s {}".format(
+            getuser(), self.hostlist_clients[0], file_name)
+        try:
+            result = subprocess.check_output(cmd, shell=True)
+        except subprocess.CalledProcessError as err:
+            self.fail("Get remote file size method failed with: {}".format(err))
+
+        return int(result)
```

Comment:
This seems like a more general/generic utility function that other tests might be able to use.  Why not put it into a _general utilities_ type library instead of it being only available to this test?